home/views.py

```diff
@@ -101,12 +114,15 @@ def dashboard(request):
             for stu_att in student_attendance:
                 stu_att_village[stu_att.student.village] += 1
             # Mehgawan Side
-            stu_att_village['MS'] = stu_att_village['M'] + stu_att_village['C'] + stu_att_village['A'] + stu_att_village['S']
+            stu_att_village['MS'] = stu_att_village['M'] + \
+                stu_att_village['C'] + \
+                stu_att_village['A'] + stu_att_village['S']
```

Comment:
Instead of `\` here, I think putting the whole expression in brackets would be better.